pkg/config/config_unix.go

```diff
@@ -38,7 +38,8 @@ func DefaultConfig() PluginConfig {
 			NoPivot:            false,
 			Runtimes: map[string]Runtime{
 				"runc": {
-					Type: "io.containerd.runc.v1",
+					Type:                 "io.containerd.runc.v1",
+					ContainerAnnotations: []string{},
```

Comment:
If we are going to show this here as an empty list, we'd also want to show PodAnnotations I think...

Reply:
This is needed for 'containerd config default' to add this item to the generated content.
Yes, we'd want it for PodAnnotations as well, but not in this PR I believe.

Reply:
yeah we don't show all of the empty elements in the default.. it's on purpose

Reply:
ok, thanks for the explanation. Should I remove it then?

Reply:
I set ContainerAnnotations to ["kubernetes.io/*", "k8s.io/*] by default. I think these are a reasonable default values. Please, review.

Reply:
for container annotations it looks like the current keys are:
```
	// TODO: change those label names to follow kubernetes's format
	podDeletionGracePeriodLabel    = "io.kubernetes.pod.deletionGracePeriod"
	podTerminationGracePeriodLabel = "io.kubernetes.pod.terminationGracePeriod"

	containerHashLabel                     = "io.kubernetes.container.hash"
	containerRestartCountLabel             = "io.kubernetes.container.restartCount"
	containerTerminationMessagePathLabel   = "io.kubernetes.container.terminationMessagePath"
	containerTerminationMessagePolicyLabel = "io.kubernetes.container.terminationMessagePolicy"
	containerPreStopHandlerLabel           = "io.kubernetes.container.preStopHandler"
	containerPortsLabel                    = "io.kubernetes.container.ports"
```
but from the todo they may change soon, let's wait for @Random-Liu to chime in.  We might need to reserve two sets of patterns this io.kubernetes one currently in use and kubernetes.io suggested by the todo?

We'll also want to go ahead and set pod annotations default keys for k8s here I think so they will pass through as well,... I mean while we're at it, why not do the same for pod annotations as we are doing for container. It's that or we could put them all in the docs for explanation and to reserve them and users could config as they like, thus keeping the default as is, but allowing annotations to be passed through for 3rd party integrations. 

Again.. I'd like to hear from some other sig-node members regarding their opinion for defaulting the passthrough of pod and container annotations.

Reply:
Changed ContainerAnnotations value to ["io.kubernetes.*", "io.k8s.*"]

In my opinion it's not that important. We can change this later on if we want to. The main purpose of this PR is to pass forgotten annotations to the runtime.

Reply:
Would it work if we remove setting defaults for *Annotations settings from this PR and make another PR that sets them? Would it help to merge this PR faster?

Reply:
> Would it work if we remove setting defaults for *Annotations settings from this PR and make another PR that sets them? Would it help to merge this PR faster?

probably yes. 

Just add in the config.md doc that *.kubernetes.* and *.k8s.* are reserved for both pod and container annotation keys.

Reply:
I don't think we need any default value here.

Reply:
I've removed default settings as @Random-Liu suggested